catalogue/webapp/test/e2e/helpers/urls.ts

```diff
@@ -0,0 +1,14 @@
+export const baseUrl = (): string => {
+  switch (process.env.NODE_ENV) {
+    case 'production':
+      return 'https://wellcomecollection.org';
+    case 'staging':
+      return 'https://www-stage.wellcomecollection.org';
+    default:
+      return `http://localhost:3000`;
+  }
+};
```

Comment:
```suggestion
export const baseUrl: string = ((env: string) => {
  switch (process.env.NODE_ENV) {
    case 'production':
      return 'https://wellcomecollection.org';
    case 'staging':
      return 'https://www-stage.wellcomecollection.org';
    default:
      return 'http://localhost:3000';
  }
})(process.env.NODE_ENV);
```

Means you won't have to call the method each time as `process.env.NODE_ENV` is never configurable.

Reply:
nice , makes sense